class ConflictError(Exception):
    pass


class ValidationError(Exception):
    pass


class NotFoundError(Exception):
    pass
